src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,67 @@
+// src/tests/components/ForecastSummary.test.js
+
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1111111,
+    description: "Stub description",
+    icon: "800",
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+  };
+  function onSelect() {}
```

Comment:
`onSelect()` is a valid prop, so it should be included as part of that object. You would also need to update how it is called in the functions below
```
  const validProps = {
    date: 1111111,
    description: "Stub description",
    icon: "800",
    temperature: {
      min: 12,
      max: 22,
    },
    onSelect: () => {},
  };
```